public/_locales/cs/messages.json

```diff
@@ -0,0 +1,49 @@
+{
+	"extensionDescription": {
+		"message": "Krásně vizualizované statistiky pro vaše e-mailové účty Thunderbird"
+	},
+	"popup": {
+		"message": "",
+		"account": "Účet | Účty",
+		"messagesInFolder": "{0} zpráv v {1} složce"
+	},
+	"stats": {
+		"message": "",
+		"mailsTotal": "Celkem e-mailů",
+		"withinYears": "do {0} let",
+		"mailsUnread": "Nepřečtené e-maily",
+		"niceWork": "Pěkná práce!",
+		"percentOfReceived": "{0}% přijatých",
+		"mailsReceived": "Přijaté e-maily",
+		"percentOfTotal": "{0}% z celkového počtu",
+		"mailsSent": "E-maily odeslány",
+		"mailsPerMonth": "E-maily za měsíc",
+		"mailsYear": "mails/year",
```

Comment:
Line 21 is still missing a translation, isn't it?

Reply:
You are correct! Thanks for catching this and for working with me.

Just to confirm, `<a href='{0}' target='_blank'>Github</a>` should be left untranslated, is that correct?

Reply:
Edits to mails per year in b644d36